hil-test/tests/rmt.rs

```diff
@@ -59,11 +60,16 @@ mod tests {
         };
 
         cfg_if::cfg_if! {
-            if #[cfg(any(feature = "esp32", feature = "esp32s2"))] {
+            if #[cfg(feature = "esp32")] {
                 let  rx_channel = {
                     use esp_hal::rmt::RxChannelCreator;
                     rmt.channel1.configure(rx, rx_config).unwrap()
                 };
+            } else if #[cfg(feature = "esp32s2")] {
+                    let  rx_channel = {
```

Comment:
Please clean up the indentation here. Also please remove one of the two spaces between `let` and `rx_channel` in all the blocks around here.

Reply:
Looks like rust-fmt is missing something here :D Thanks!

Reply:
rustfmt very rarely touches macros :)